examples/README.md

```diff
@@ -5,3 +5,19 @@ The Zarf examples demonstrate different ways to utilize Zarf in your environment
 To test create a virtual area to test all examples, you can run `make all` or `make vm-init` if you've already run the examples before. Run `make vm-destroy` to clean up.
 
 > **NOTE**: Examples are not meant for production use, they exist to demonstrate how to use Zarf in various ways.  Also, examples utilize software pulled from multiple sources and _some_ of them require authenticated access. Check the examples themselves for the specific accounts / logins required.
+
+
+&nbsp;
+
+
+| Example                                                          |      Description      |
```

Comment:
- [ ] Make both labels left aligned 
- [ ] Add language to note about examples being unsuported

Reply:
@Madeline-UX  Are you asking for a note in each example description or is adding to the note above the chart good enough?

Reply:
Hmmm - For this pull request I think just above is fine but you bring up a good point about looking at each example